.github/workflows/dockerize.yml

```diff
@@ -0,0 +1,30 @@
+name: Update docker image
+
+on:
+  push:
+  workflow_dispatch:
+env:
+  IMAGE_NAME: neonlabsorg/openzeppelin-contracts
+jobs:
+  dockerize:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Define tag
+        id: define-env
+        run: |
+          if [[ "${{ github.ref_name }}" == 'master'  ]]; then
+            tag='latest'
+          else
+            tag='${{ github.sha }}'
```

Comment:
what if will branch? will be good to see a branch in tag :)

Reply:
Changed commit hash to the branch